.github/PULL_REQUEST_TEMPLATE/tep.md

```diff
@@ -0,0 +1,15 @@
+---
+name: TEP
+about: Create a TEP pull-request
+---
+
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Summary
+
+<!--
+Summarize your TEP here - ideally you can get that straight from your
+descriptive commit message or the summary of your TEP.
```

Comment:
would it make sense to include a link to our TEP docs?

Reply:
Indeed 🙃